doc/project/project_roles.rst

```diff
@@ -314,15 +314,46 @@ Release Activity
 Merge Criteria
 ++++++++++++++
 
-* All continuous integration checks have passed
+* Minimal of 2 approvals from the list below, including an approval by the
+  designated assignee:
+
+  * Collaborators from the same subsystem.
+  * Maintainers and collaborators of other areas.
```

Comment:
> Maintainers and collaborators of other areas.

Is this meant to say "Maintainers or collaborators of other areas affected by the PR" ?

I'm wondering if these 3 sentences would not be better rephrased overall to something like:

```
* Minimal of 2 approvals.
* PRs should be reviewed by at least a maintainer or collaborator of each affected area; 
  Unless the changes to a given area are considered trivial enough, in which case
  approvals by other affected subsystems maintainers/collaborators would suffice.
```